File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.swift
// Single-line comment
/* Multi-line
   comment */

import Foundation

@MainActor
class ViewModel: ObservableObject {
    @Published var count: Int = 0

    func fetch() async throws {
        let (data, _) = try await URLSession.shared.data(from: url)
        let decoded = try JSONDecoder().decode(Response.self, from: data)
        await MainActor.run {
            self.results = decoded
        }
    }
}

struct Calculator {
    subscript(index: Int) -> Int {
        return index * 2
    }
}

let closure = { [weak self] (num: Int) -> String in
    return "Number: \(num)"
}

File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.py
# Single-line comment
"""Multi-line
   string/comment"""

def factorial(n: int) -> int:
    if n <= 1:
        return 1
    else:
        return n * factorial(n-1)

class MyClass(SuperClass):
    class_var = 10

    def __init__(self):
        self._protected = True
        self.__private = False

@decorator
async def fetch_data():
    async with session.get(url) as response:
        data = await response.json()

f_string = f"Value: {x}"
list_comp = [x**2 for x in range(10)]
lambda_fn = lambda x: x * 2
